Server/BranchingStrategy/ConflictLocator.cs

```diff
@@ -20,13 +20,35 @@ public async Task<IReadOnlyList<IdentifiedConflict>> FindConflictsWithin(IReadOn
 	{
 		var results = new List<IdentifiedConflict>();
 		var context = await ConstructContext();
+
+		await GatherInitialResolutions(branches, context);
+
 		for (var i = 0; i < branches.Count - 1; i++)
 		{
 			results.AddRange(await FindConflictsBetween([branches[i]], branches.Skip(i + 1).ToArray(), context));
 		}
 		return results.ToArray();
 	}
 
+	private static async Task GatherInitialResolutions(IReadOnlyList<string> branches, ConflictLocatorContext context)
```

Comment:
Why is this async? Even the TODO should still be sync.

Reply:
Yeah, that's why `await Task.Yield()` is in place - that's where we'd await for gathering additional information.